Editor/Inspector/Menu/MenuItemGUI.cs

```diff
@@ -215,11 +220,46 @@ public void DoGUI()
             EditorGUILayout.BeginHorizontal();
 
             EditorGUILayout.BeginVertical();
-            EditorGUI.BeginChangeCheck();
-            EditorGUILayout.PropertyField(_name, G("menuitem.prop.name"));
-            if (EditorGUI.EndChangeCheck())
+
```

Comment:
One note: Please make sure this doesn't break in some terrible way when multiple items are selected...

Reply:
It's maybe better to multi-edit the label field in that case. Seems like less surprising behavior than editing the name of all the GameObjects at once?

Reply:
Should be current behaviour as of the latest commit